collect_app/src/main/java/org/odk/collect/android/application/initialization/upgrade/BeforeProjectsInstallDetector.kt

```diff
@@ -0,0 +1,22 @@
+package org.odk.collect.android.application.initialization.upgrade
+
+import android.content.Context
+import androidx.preference.PreferenceManager
+import org.odk.collect.android.utilities.FileUtils
+import org.odk.collect.upgrade.InstallDetector
+import java.io.File
+
+class BeforeProjectsInstallDetector(private val context: Context) : InstallDetector {
```

Comment:
I think it would be good to have a comment that explains what that `BeforeProjectsInstall` means. Now we know it but in the future who knows... you can explain it a bit and include version name that was the first one with projects.